docs/source/user_guide/gcm_based_inference/introduction.rst

```diff
@@ -170,5 +170,56 @@ This intervention says: "I'll ignore any causal effects of X on Y, and set every
 to 2.34." So the distribution of X will remain unchanged, whereas values of Y will be at a fixed
 value and Z will respond according to its causal model.
 
+These are the basic steps that need to happen. While we can run these steps explicitly, often they get
+executed as part of other steps, e.g. when fitting and re-fitting as part of computing confidence
+intervals. The next section therefore dives into a more typical usage pattern of the ``dowhy.gcm`` package.
+
+Typical usage of the ``dowhy.gcm`` package
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In practice, we may not execute the steps we've learned above explicitly and in this order, but they
+get called by other abstractions. E.g. we usually want to use confidence intervals, when answering
+causal questions, to quantify our confidence in the results. In this case, ``fit`` will be called on
+behalf of us, but we won't call it directly.
+
+At the same time, we may introduce some optional steps, again to improve our confidence in the
+results, such as some basic checks to see whether the independence structure of the graph is
+consistent with the data.
+
+Modeling an SCM and validating the structure of its causal graph against the data
```

Comment:
We don't do the validation in the introduction anymore, right?

Reply:
Removed that paragraph.